src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php

```diff
@@ -75,9 +75,9 @@ public function request(Request $request, MailerInterface $mailer): Response
 <?php } ?>
     public function checkEmail(): Response
     {
-        // We prevent users from directly accessing this page
+        // Generate a fake token - the user does not exist or someone hit this page directly.
         if (null === ($resetToken = $this->getTokenObjectFromSession())) {
-            return $this->redirectToRoute('app_forgot_password_request');
+            $resetToken = $this-resetPasswordHelper->generateFakeResetToken();
```

Comment:
We can only do this if the method exists. Honestly, we should probably just ensure that the user is using a new enough version of reset bundle. I'd rather say "Hey! You need to upgrade!" vs generating the old code... and then they don't have the better code even after they upgrade reset later.

Reply:
agreed and done